lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -302,6 +338,59 @@ END:
     type: process
     lambda: build-client-oauth-response
 
+CRI_TICF:
+  response:
+    type: process
+    lambda: call-ticf-cri
+    lambdaInput:
+      journeyType: main
```

Comment:
(Just a comment) Outside the scope of the ticket, but it could be nice to use this input to reduce the number of separately-defined states. We could have more events, and the journeyType being main/ reuse/ no-match/ another-way can define the event outputted.

That being said, it would hide some information from the journey map which we can now visualise well